src/content/docs/troubleshooting.mdx

```diff
@@ -237,6 +237,11 @@ This can be caused by a number of things. The most common causes are:
    use caution when publishing patches that include changes to native code from
    plugins. In the worst case, these changes may cause your patched app to
    crash.**
+   :::tip
+   Add your `pubspec.lock` file to version control so you're always aware of
```

Comment:
Dart has a great guide about the lock file, should we link it too?

https://dart.dev/guides/libraries/private-files#pubspec-lock

Reply:
If there were a Flutter-specific guide, I'd say yes. The dart guide is a little less clear/direct than I'd like (basically, I just want to tell users "Yes, check it in" rather than "don't check it in _unless_ your package is an application package" and then having the users wonder what an application package is and whether their Flutter app is one of those, etc.). I'll copy some of the text from those docs over.

Reply:
Sounds good, my suggestion was more in the lines that this guide corroborate for what we are suggesting the users.

But no strong opinions, the content is already good!